recipes/cpnest/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cpnest" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d698aa9ebf924a6a5e54c29165b75cf67f38616179256495e02e4e75391c718f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
I generated this recipe with `conda skeleton pypi` but it was a few months ago, I presume the skeleton has been updated since then; will change.

Reply:
The skeleton for ``pypi`` was updated but I don't think there is a new release with the modifications which will fix this part.
Thanks for changing it! :)